runtime/oti/j9consts.h

```diff
@@ -767,8 +767,8 @@ extern "C" {
 #define J9_JNI_OFFLOAD_SWITCH_JVMTI_MONITOR_WAITED						0x2E
 #define J9_JNI_OFFLOAD_SWITCH_JVMTI_MONITOR_CONTENDED_ENTER				0x2F
 #define J9_JNI_OFFLOAD_SWITCH_JVMTI_MONITOR_CONTENDED_ENTERED			0x30
-/* 0x31 Reserved for JVMTI event */
-/* 0x32 Reserved for JVMTI event */
+#define J9_JNI_OFFLOAD_SWITCH_J9JVMTI_VIRTUAL_THREAD_MOUNT              		0x31
+#define J9_JNI_OFFLOAD_SWITCH_J9JVMTI_VIRTUAL_THREAD_UNMOUNT            		0x32
```

Comment:
Inconsistent spacing should be removed. The value can be separated just by a single space. This can be applied to the other `J9_JNI_OFFLOAD_SWITCH_JVMTI_` macros from line 743 to 787.
```suggestion
#define J9_JNI_OFFLOAD_SWITCH_J9JVMTI_VIRTUAL_THREAD_MOUNT 0x31
#define J9_JNI_OFFLOAD_SWITCH_J9JVMTI_VIRTUAL_THREAD_UNMOUNT 0x32
```

Reply:
removed inconsistent spacing, only one space after define, and changed existing macros from line 743 to 787.